apps/settings/src/components/UserList/UserRow.vue

```diff
@@ -219,7 +219,7 @@
 					</ActionButton>
 				</Actions>
 				<div v-click-outside="hideMenu" class="userPopoverMenuWrapper">
-					<div class="icon-more"
+					<button class="icon-more"
 						@click="toggleMenu" />
```

Comment:
In case it does reload, something like that should work:

```suggestion
					<button class="icon-more"
						@click.prevent="toggleMenu" />
```

https://vuejs.org/v2/guide/events.html#Event-Modifiers

Reply:
> I think the default behaviour when clicking a button is to send a form. But that might only be the case when <button> is inside a <form>

It indeed only happens in a form. There is (thankfully :grin:) no default action for `<button>`.

But why not. Better safe than sorry.
Rebased on master and compiled.